tools/devworkspace-generator/package.json

```diff
@@ -46,7 +46,8 @@
     "inversify": "^5.0.1",
     "js-yaml": "^4.0.0",
     "jsonc-parser": "^3.0.0",
-    "reflect-metadata": "^0.1.13"
+    "reflect-metadata": "^0.1.13",
+    "@eclipse-che/che-devworkspace-generator": "7.75.0-SNAPSHOT"
```

Comment:
@mkuznyetsov why do we need this dependency here?
And as I see "@eclipse-che/che-devworkspace-generator": "7.75.0-SNAPSHOT" is not published https://www.npmjs.com/package/@eclipse-che/che-devworkspace-generator?activeTab=versions

Reply:
yep, it is a mistake, working on a fix